src/components/navbar/Navbar.js

```diff
@@ -16,18 +16,17 @@ export const Navbar = () => {
         zIndex: theme.zIndex.drawer + 1
     }}>
         <Toolbar>
-            <div style={{marginLeft: "2%", marginRight: "1%"}}>
+            <div style={{marginTop: '-5px'}}>
                 <Link to="/" style={{color: "white", textDecoration: "none"}}>
-                    <Typography variant="h6">
-                        e-Stella
-                    </Typography>
+                    <img src="https://imgur.com/5LUwfu8.png" alt="logo" width="30%" />
                 </Link>
             </div>
-            <div style={{marginLeft: "1%", marginRight: "auto"}}>
+            <div style={{marginLeft: '-30%'}}>
```

Comment:
Wiem że blackWeek za niedługo ale to naprawdę tak ma być?

Reply:
Tak, bo po dodaniu loga rozwalił się cały navbar. W sensie zrobił się flex, który rozłożył elementy po równo i nijak nie wiem inaczej, jak to naprawić. Mogę jeszcze trochę poszperać, bo to faktycznie teraz lata, ale nie jestem pewny, czy coś się z tym da zrobić XDD